src/core/base.js

```diff
@@ -45,6 +45,10 @@ getJasmineRequireObj().base = function(j$, jasmineGlobal) {
     return env;
   };
 
+  j$.getPromise = function() {
+    return j$.getEnv().getPromise() || j$.getGlobal().Promise;
+  };
+
```

Comment:
I don't think that the configured promise library needs to be (or should be) exposed back outside of Jasmine itself. If Jasmine has been told to use whichever library I specify, I shouldn't need to check up on it to make sure.